app/Core/ExternalTask/ExternalTaskManager.php

```diff
@@ -48,6 +48,8 @@ public function getProvider($name)
     public function getProvidersList()
     {
         $providers = array_keys($this->providers);
-        return array_combine($providers, $providers);
+        if (count($providers)) {
+            return array_combine($providers, $providers);
+        }
     }
```

Comment:
Hi Frédéric,
you are so right, sorry, i've overlooked!
Thank you and have a happy new year!